share/ci/install/clang.sh

```diff
@@ -11,7 +11,10 @@ clang_version=$(echo $CXX_VERSION | tr -d "clang++-")
 echo "Clang-version: $clang_version"
 
 if ! agc-manager -e clang@${clang_version}; then
-    apt install -y clang-${clang_version}
+    apt install -y clang-${clang_version} libomp-${clang_version}-dev
+    if [[ "$PIC_BACKEND" =~ "omp2b.*" ]] ; then
+        apt install -y libomp-${clang_version}-dev
```

Comment:
This is the same as the second package two lines above?

Reply:
I forgot to remove libomp from the first install command. I fixed it now. It looks like if I remove the first apt repository in llvm.list all is working fine.